package/yast2-security.spec

```diff
@@ -32,12 +32,14 @@ BuildRequires:  perl-XML-Writer
 BuildRequires:  update-desktop-files
 BuildRequires:  yast2-pam
 BuildRequires:  yast2-devtools >= 4.2.2
-# Y2Security::SelinuxConfig requires Yast::Bootloader
+# Y2Security::Selinux requires Yast::Bootloader
 BuildRequires:  yast2-bootloader
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:yast-rake) >= 0.2.5
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
 # CFA::SysctlConfig
 BuildRequires:  yast2 >= 4.2.66
+# CFA::Selinux
+BuildRequires:  augeas-lenses
```

Comment:
Realizing now that it is actually not needed because of the `yast2 >= 4.2.66` requirement (https://github.com/yast/yast-yast2/blob/64ff8b58e57ae84448f099ff4f97b0d92b76bfeb/package/yast2.spec), although it does not hurt.

Reply:
yep, better specify it in each package so we do not depend on others and tight with them